playbook/app/pb_kits/playbook/pb_bar_graph/barGraphSettings.js

```diff
@@ -1,5 +1,5 @@
 import { commonSettings } from '../pb_dashboard/commonSettings.js'
-import typography from '../tokens/exports/_typography.scss'
+import typography from '../tokens/exports/_typography.module.scss'
```

Comment:
Specifying `.module.scss` for a given Sass file will enable the file to use CSS modules.